.github/actions/submit-test/action.yml

```diff
@@ -0,0 +1,179 @@
+# Copyright 2024 The Ground Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: Submit to survey
+
+inputs:
+  android-repository:
+    description: 'ground-android repository under test'
+    default: sufyanAbbasi/ground-android
```

Comment:
Did you mean to update this to google/?

Does this also work in forks?

Reply:
> Did you mean to update this to google/?

Whoops, this is for testing purposes since the main repo doesn't have the actions defined yet. Also updated in the other PR.

> Does this also work in forks?

If you want to end to end with two forks, then no, but most likely you'll want to test the currently forked repo against the prod other repo, aka the google one.